extensions/azurecore/src/test/azureResource/tree/treeProvider.test.ts

```diff
@@ -108,7 +107,7 @@ describe('AzureResourceTreeProvider.getChildren', function(): void {
 		should(children[0]).instanceof(AzureResourceAccountNotSignedInTreeNode);
 	});
 
-	xit('Should handle errors.', async function(): Promise<void> {
+	xit('Should handle errors.', async function (): Promise<void> {
```

Comment:
Ah didn't know this was disabled, let me see if I can get this enabled too (and see if there is any reason to get it enabled)